mod-infra/src/main/scala/at/forsyte/apalache/infra/ExitCodes.scala

```diff
@@ -0,0 +1,57 @@
+package at.forsyte.apalache.infra
+
+import tlc2.output.EC
+
+/**
+ * Exit codes that are reported by Tool.
+ *
+ * We are using a small subset of exit codes. Although we maintain exit codes that are compatible with TLC, we believe
+ * that exit codes should not replace richer error explanations.
+ *
+ * TLC currently specifies exit codes
+ *   - 0 for success,
+ *   - 10–14 for (various kinds of) property violations,
+ *   - 75–77 for "failures" (e.g., properties that are tautologies, or primed variables in invariants), and
+ *   - 150–153 and 255 for syntax errors and system-level errors.
+ *
+ * In addition, Apalache uses exit code
+ *   - 120 for type-checking errors.
```

Comment:
Seems like this is now missing some of the exit codes we use in Apalache?

Reply:
I think I got it now, reworded in 3841307ae.